### PR TITLE
guild check

### PR DESCRIPTION
**Pull request by phenom4n4n**
Adds guild checks to the commands that need them and also randomizes embed colors since that looks better than black.

### CODE REVIEW

File: conversationgames/conversationgames.py [hunk 1]
```diff
@@ -717,7 +718,7 @@ async def wouldyourather(self, ctx):
         embed = discord.Embed()
         embed.title = "Would you rather.."
         embed.description = strings[i]
-        await ctx.send(embed=embed)
+        await ctx.send(embed=randomize_colour(embed))
```
Comment (lionirdeadman) [inline conversationgames/conversationgames.py]: I think the randomize_colour shouldn't be used, can you remove that from the PR?

**Review by lionirdeadman — CHANGES_REQUESTED** [review]
Otherwise looks good, will need to test.

**Comment by phenom4n4n** [comment]
Done.

**Review by lionirdeadman — APPROVED** [review]

**Comment by lionirdeadman** [comment]
Thanks!

**Pull request merged**